<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddColumnWindow</class>
 <widget class="QDialog" name="AddColumnWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>577</width>
    <height>302</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dodaj kolumnę</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>250</y>
     <width>171</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_colName">
   <property name="geometry">
    <rect>
     <x>282</x>
     <y>60</y>
     <width>241</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>134</x>
     <y>60</y>
     <width>141</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Nazwa nowej kolumny</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>134</x>
     <y>90</y>
     <width>121</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Typ nowej kolumny</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_colType">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>90</y>
     <width>241</width>
     <height>24</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>INTEGER</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>TEXT</string>
    </property>
   </item>
  </widget>
  <widget class="QCheckBox" name="checkBox_unique">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>150</y>
     <width>111</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Pole unikalne</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_primaryKey">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>150</y>
     <width>111</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Klucz główny</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>AddColumnWindow</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>AddColumnWindow</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
